test/helpers.dart

```diff
@@ -18,3 +19,11 @@ extension WidgetTesterExtension on WidgetTester {
     await pumpAndSettle();
   }
 }
+
+extension CommonFindersX on CommonFinders {
```

Comment:
Hah never seen X been used as a short form of Extension before, is this common? I think I'd prefer the word written out.

Reply:
Yeah, I didn't know either, but seems to be quite common.